rust/src/ostree_diff.rs

```diff
@@ -104,12 +104,10 @@ fn diff_recurse(
                     diff.changed_files.insert(path);
                 }
             }
+        } else if is_dir {
```

Comment:
I'm OK with this but I feel the previous code was logically clearer (if addition else deletion) instead of mixing them.  But again fine as is.

Reply:
Will revert as that was my feeling as well and this is a nop.